readme.txt

```diff
@@ -105,105 +105,87 @@ There are opportunities for developers at all levels to contribute. [Learn more
 
 = 4.7 =
 
-* Release date:
-* Release post:
+**Performance Improvements**
 
-**New Features and Improvements**
+* Experience better performance with single page load caching of the media summary.

```

Comment:
I think I would remove this line.
It's really the same as the second item, `* We made some improvements to avoid slow queries on sites with thousands of images.`, then they both apply to OG tags.